ui/app/adapters/pki/config.js

```diff
@@ -1,22 +1,28 @@
+import { assert } from '@ember/debug';
 import { encodePath } from 'vault/utils/path-encoding-helpers';
 import ApplicationAdapter from '../application';
 
 export default class PkiConfigAdapter extends ApplicationAdapter {
   namespace = 'v1';
 
   urlForCreateRecord(modelName, snapshot) {
-    const { backend, formType, type } = snapshot.record;
+    const { backend, type } = snapshot.record;
+    const { formType, useIssuer } = snapshot.adapterOptions;
     if (!backend || !formType) {
       throw new Error('URL for create record is missing required attributes');
     }
-    const baseUrl = `${this.buildURL()}/${encodePath(backend)}/issuers`;
+    const baseUrl = `${this.buildURL()}/${encodePath(backend)}`;
     switch (formType) {
+      case 'import':
+        return useIssuer ? `${baseUrl}/issuers/import/bundle` : `${baseUrl}/config/ca`;
       case 'generate-root':
-        return `${baseUrl}/generate/root/${type}`;
+        return useIssuer ? `${baseUrl}/issuers/generate/root/${type}` : `${baseUrl}/root/generate/${type}`;
```

Comment:
Not sure if we want to refactor this after our discussion with Alex? But I think it makes sense for the fallback to be the "old" endpoint of `/root/generate`

Reply:
Yeah, I'll update the model capabilities to check instead for permissions on the issuer endpoints